models/coded.go

```diff
@@ -40,9 +40,9 @@ func (c *Coded) AddCodeIfPresent(codeElement xml.Node) {
 
 	//extract nullFlavor from attribute if it is NA
 	nullFlavorAttribute := codeElement.Attribute("nullFlavor")
-	nullFlavorValueSet := codeElement.Attribute("valueSet")
-	if nullFlavorAttribute != nil && nullFlavorValueSet != nil {
-		code = nullFlavorValueSet.String()
+	nullFlavorValueSet := codeElement.Attributes()
```

Comment:
How come we are pulling out all the attributes and referencing the `valueSet` attribute later? Why not just extract the one we need on the codeElement.Attribute() call?